nilearn/glm/tests/test_first_level.py

```diff
@@ -1,8 +1,5 @@
-"""
-Test the first level model.
-"""
+"""Test the first level model."""
```

Comment:
Redundant comment. Filename says it all.

Reply:
Agreed.

Some of those comments I think I introduced to keep flake8 happy but #3534 should make things easier.

Reply:
done